app/views/documentation.scala.html

```diff
@@ -458,15 +458,23 @@ <h4>Maven Example (<a href="https://github.com/webjars/sample-jetty_war">example
 
                 <h4>Making dependencies version agnostic</h4>
 
-                When using Spring Framework version 4.2 or higher, it will automatically detect the <code>webjars-locator</code> library on the classpath and use it to automatically resolve the version of any WebJar assets for you.
-                In order to enable this feature, you will need to add the webjars-locator library as a dependency of your application in the <span class="label label-info">pom.xml</span> file, like:
+                When using Spring Framework version 4.2 or higher, it will automatically detect the <code>webjars-locator-core</code> library on the classpath and use it to automatically resolve the version of any WebJar assets for you.
+                In order to enable this feature, you will need to add the webjars-locator-core library as a dependency of your application in the <span class="label label-info">pom.xml</span> file, like:
                 <pre><code>&lt;dependencies&gt;
     &lt;dependency&gt;
         &lt;groupId&gt;org.webjars&lt;/groupId&gt;
-        &lt;artifactId&gt;webjars-locator&lt;/artifactId&gt;
+        &lt;artifactId&gt;webjars-locator-core&lt;/artifactId&gt;
         &lt;version&gt;0.30&lt;/version&gt;
     &lt;/dependency&gt;
 &lt;/dependencies&gt;</code></pre>
+                <br>
+                And extend your XML config with a resource-chain element:
```

Comment:
I'm not totally clear on when this is required.  I think at least in Spring Boot, the WebJars resource handler is added automatically when `webjars-locator-core` is on the classpath.  Maybe we need to provide more details on the differences between Boot & vanilla Spring.